telemetry/admin-metrics-export/src/test/scala/io/buoyant/telemetry/admin/AdminMetricsExportTelemeterInitializerTest.scala

```diff
@@ -0,0 +1,38 @@
+package io.buoyant.telemetry.admin
+
+import com.twitter.finagle.Stack
+import com.twitter.finagle.util.LoadService
+import io.buoyant.admin.Admin
+import io.buoyant.config.Parser
+import io.buoyant.telemetry.{TelemeterConfig, TelemeterInitializer}
+import io.buoyant.test.FunSuite
+
+class AdminMetricsExportTelemeterInitializerTest extends FunSuite {
+
+  test("io.l5d.admin telemeter loads") {
+    val yaml =
+      """|kind: io.l5d.admin
+         |""".stripMargin
```

Comment:
these tests weren't even being run because I forgot to update the build file hahahaha whoooooooops